### PR TITLE
Combo Update

### PR DESCRIPTION
**Pull request by IGSvt**
-Replace "Simple Input" with Box Input Type
-Add missing information regarding Figma 
-Update references 
-Edit current information

### CODE REVIEW

File: en/components/combo.md [hunk 1]
```diff
@@ -6,59 +6,52 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
-## Input Type
+## Combo Input Type
 
-The Combo input provides selection between a line type for a more airy style, border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
+The Combo provides selection between a border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
 
-<img class="responsive-img" src="../images/combo_demo.png" srcset="../images/combo_demo@2x.png 2x" />
-`line`
 
 <img class="responsive-img" src="../images/combo_border.png" srcset="../images/combo_border@2x.png 2x" />
```
Comment (imincheva) [inline en/components/combo.md:22]: The types are swapped. On the border type it says box and on the box type it says border

File: en/components/combo.md [hunk 2]
```diff
@@ -6,59 +6,52 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
-`border`
 
 <img class="responsive-img" src="../images/combo_box.png" srcset="../images/combo_box@2x.png 2x" />
-`box`
 
-## Interaction State
+## Enabled and Disabled States
 
-Each of the three Combo input types can be inserted in an enabled or disabled state.
+Each of the two Combo input types can be inserted in an enabled or disabled state. In Figma the input can be disabled from the property panel.
```
Comment (imincheva) [inline en/components/combo.md:28]: Each of the two -> Both
property panel -> properties panel

File: en/components/combo.md [hunk 3]
```diff
@@ -6,59 +6,52 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
 
 <img class="responsive-img" src="../images/combo_enabledstate.png" srcset="../images/combo_enabledstate@2x.png 2x" />
```
Comment (imincheva) [inline en/components/combo.md:30]: You can add "Enabled" and "Disabled" as titles on the images, like you did on the rest of them - (Idle, Filled etc.)

File: en/components/combo.md [hunk 4]
```diff
@@ -6,59 +6,52 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
-`filled`
 
 Every experienced designer uses constraints wisely to limit the user input and avoid invalid states, hence the availability of validation styles. Through the available validation styles, the Combo is consistent with the standard [Input](input.md) and equipped for sophisticated designs that display success, warning, and error visuals.
 
 <img class="responsive-img" src="../images/combo_success.png" srcset="../images/combo_success@2x.png 2x" />
 <img class="responsive-img" src="../images/combo_warning.png" srcset="../images/combo_warning@2x.png 2x" />
 <img class="responsive-img" src="../images/combo_error.png" srcset="../images/combo_error@2x.png 2x" />
 
-In Sketch, the states and validation states are achieved with `Symbol Overrides`, while in Adobe XD we are using the `Component States` paradigm to let you easily switch between them.
+In Sketch, the states and validation states are achieved with `Symbol Overrides`, while in Figma and Adobe XD we are using the `Component States` paradigm to let you easily switch between them.
```
Comment (imincheva) [inline en/components/combo.md:49]: I'm not sure whether its appropriate to say that in Figma we use the "Component States" paradigm like in Adobe XD. I think it's better to say we have a property for this, which can be found on the properties panel or something similar.

File: en/components/combo.md [hunk 5]
```diff
@@ -6,59 +6,52 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
 
 > [!Note]
 > The Combo input has only one layout option, but this override has been preserved in Sketch to ensure consistency with the standard [Input](input.md).
 
-## Dropdown Type
+## Drop down Type
 
 The dropdown used for the Combo comes with the appropriate sizing for desktop and mobile use similarly to the normal [Dropdown](dropdown.md) component.
```
Comment (imincheva) [inline en/components/combo.md:56]: I see you changed dropdown to drop down. You can do it here too, as well as the rest of the places, where it still says dropdown.

File: en/components/combo.md [hunk 6]
```diff
@@ -6,59 +6,52 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
-The Combo has two pieces: an input containing the current selection and a dropdown that is shown in order to select one or more items.
+The Combo has two pieces: an input containing the current selection and a drop down that is shown in order to select one or more items.
 
 <img class="responsive-img" src="../images/combo_demo.png" srcset="../images/combo_demo@2x.png 2x" />
 
-## Input Type
+## Combo Input Type
 
-The Combo input provides selection between a line type for a more airy style, border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
+The Combo provides selection between a border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
```
Comment (imincheva) [inline en/components/combo.md:19]: The comma after background is unnecessary.

File: en/components/combo.md (hunk 3 above)
Comment (IGSvt) [reply en/components/combo.md:30]: This is not relevant anymore

File: en/images/combo_filled.png
Comment (imincheva) [inline en/images/combo_filled.png:1]: Value should say Austria, Europe.

File: en/images/combo_filled@2x.png
Comment (imincheva) [inline en/images/combo_filled@2x.png:1]: Value should say Austria, Europe

**Review by imincheva — CHANGES_REQUESTED** [review]

**Review by imincheva — DISMISSED** [review]

**Pull request merged**